input/lightcurves/DESY-LC-Archive_to_FITS_local.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import pyfits
```

Comment:
Is this script `DESY-LC-Archive_to_FITS_local.py` still used?
If no, please remove it from this PR.
If yes, please change `import pyfits` -> `from astropy.io import fits`

(`pyfits` was moved to `astropy.io.fits` about 5 years ago and is no longer maintained as `pyfits`, I don't have `pyfits` installed.)